File: ForLoops/CountingMachineRevisited/CountingMachineRevisited.java
import java.util.Scanner;
public class CountingMachineRevisited
{
  public static void main(String[]args)
  {
    Scanner keyboard = new Scanner(System.in);
    System.out.print("Count from: ");
    int cFrom = keyboard.nextInt();
    System.out.print("Count to: ");
    int cTo = keyboard.nextInt();
    System.out.print("Count by: ");
    int cBy = keyboard.nextInt();

    for(int i=cFrom; i<=cTo; i=i+cBy)
    {
      System.out.print(i + " ");
    }
  }
}

File: c++/CountingMachineRevisited/CountingMachineRevisited.cpp
#include<iostream>
using namespace std;
int main()
{
    int cFrom, cTo, cBy;
    cout<<"Count from: ";
    cin>>cFrom;
    cout<<"Count to: ";
    cin>>cTo;
    cout<<"Count by: ";
    cin>>cBy;

    for(int i=cFrom; i<=cTo; i=i+cBy)
    {
        cout<<i<<" ";
    }
}